src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java

```diff
@@ -85,15 +85,15 @@
     private String gpOutboundQueueName;
 
     @Getter
-    @Value("classpath:edifact/registration.dat")
+    @Value("classpath:edifact/pathology.dat")
     private Resource edifactResource;
 
     @Getter
-    @Value("classpath:edifact/registration.json")
+    @Value("classpath:edifact/pathology.json")
     private Resource fhirResource;
 
     @Getter
-    @Value("classpath:edifact/registration_recep.dat")
+    @Value("classpath:edifact/pathology_recep.dat")
```

Comment:
Should we rename this to `pathology_nhsack.dat` in advance of NHSACKs being implemented? 

Not a blocker, but just a discussion point

Reply:
the renaming of recep to nhsack is being done in https://gpitbjss.atlassian.net/browse/NIAD-1063